apps/frontend/dashboard/src/styles.css

```
@@ -284,14 +252,16 @@
 @custom-variant light (&:where(.light, .light *));
 
 :root {
-	--font-geist-sans: "Open Runde", ui-sans-serif, system-ui, sans-serif;
+	--font-geist-sans: var(--font-open-runde);
 	font-family:
-		"Open Runde", ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont,
-		"Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif;
+		var(--font-open-runde), ui-sans-serif, system-ui, -apple-system,
+		BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans",
+		sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)